packages/editor-ui/src/components/TagsManager/TagsView/TagsTable.vue

```diff
@@ -171,7 +171,7 @@ export default Vue.extend({
 
 		focusOnInput(): void {
 			setTimeout(() => {
-				const input = this.$refs.nameInput as any; // tslint:disable-line:no-any
+				const input = this.$refs.nameInput as any;
```

Comment:
is there a followup to clean up any references? would have preferred to keep `no-any` rule.. why did we remove it?

Reply:
> is there a followup to clean up any references?

What do you mean by references? All the `tslint:disable-*` exemptions were removed from `editor-ui`.

> would have preferred to keep `no-any` rule.. why did we remove it?

As listed above `no-any` is covered by `no-explicit-any`. This is among the [multiple exemptions](https://github.com/n8n-io/n8n/blob/efa4c567579d88f9ce764c535dfb41e7391a1286/packages/editor-ui/.eslintrc.js#L31) to be enabled in this package. Do you want to keep TSLint with `no-any` until we can enable `no-explicit-any`? Bear in mind `no-any` is already exempted inline multiple times in master.

Reply:
by references, I meant the `any`.. ya would rather keep it to avoid adding more `any` until it's enabled.. think of how much worse it could be without this rule at all

Reply:
> by references, I meant the any

Fixing poor typing is out of scope of this PR, will have to be a dedicated effort.

> would rather keep it

Enabled `no-explicit-any` and exempted inline, so TSLint can be removed.